warp/cli/warp_cli.py

```diff
@@ -92,6 +106,10 @@ def deploy(program):
 @warp.command()
 @click.argument("tx_id", nargs=1, required=True)
 def status(tx_id):
+    """
+    TX_ID: The transaction ID printed to stdout after invoking/deployment.
```

Comment:
This should probably be renamed to "tx_hash", because these are hashes now on StarkNet and are called such.